### ISSUE TITLE
about underfitting

### ISSUE THREAD

**Issue by lhj-git** [LaVi-Lab/VG-LLM#47]
Thanks for your excellent work! I uniformly sampled 10 scenes from the training set and test set in ScanRefer. I found that the checkpoint you provided performs well on the test set（MIOU 0.40） but underperforms on the training set（MIOU 0.09） , showing signs of underfitting. What might be the reasons for this?
And I am wondering why the epoch of your training strategy is set to 1.
